Sources/MockoloFramework/Models/ParsedEntity.swift

```diff
@@ -100,12 +100,13 @@ final class EntityNodeSubContainer {
 }
 
 /// Contains arguments to annotation
-/// e.g. @mockable(module: prefix = Foo; typealias: T = Any; U = String; rx: barStream = PublishSubject; history: bazFunc = true)
+/// e.g. @mockable(module: prefix = Foo; typealias: T = Any; U = String; rx: barStream = PublishSubject; history: bazFunc = true, referenceType: someVar = weak)
 struct AnnotationMetadata {
     var module: String?
     var typeAliases: [String: String]?
     var varTypes: [String: String]?
     var funcsWithArgsHistory: [String]?
+    var referenceTypes: [String: ReferenceType]?
```

Comment:
Call this 'modifiers'. The 'reference' makes sense for 'weak' but not for others like 'dynamic'.